apps/console/src/extensions/configs/application.tsx

```diff
@@ -353,7 +383,57 @@ export const applicationConfig: ApplicationConfig = {
                 return 4; // Anything else
             }
         },
-        getTabExtensions: (): ResourceTabPaneInterface[] => [],
+        getTabExtensions: (
+            props: Record<string, unknown>, 
+            features: FeatureConfigInterface
+        ): ResourceTabPaneInterface[] => {
+            const extendedFeatureConfig: ExtendedFeatureConfigInterface = features as ExtendedFeatureConfigInterface;
+            const applicationRolesFeatureEnabled: boolean = extendedFeatureConfig?.applicationRoles?.enabled;
+            const apiResourceFeatureEnabled: boolean = extendedFeatureConfig?.apiResources?.enabled;
+
+            const application: ApplicationInterface = props?.application as ApplicationInterface;
+    
+            const onApplicationUpdate: () => void = props?.onApplicationUpdate as () => void;
+
+            const tabExtensions: ResourceTabPaneInterface[] = [];
+
+            const isChoreoApp: boolean = isChoreoApplication(application);
```

Comment:
This check doesn't need for Identity Server. we can remove the `Choreo` references.

Reply:
Fixed with 20d21b8b65dda1889385e3f96ec60e4b05147013